packages/next-templates/src/app/meldingen/verhuizing-doorgeven/page.tsx

```diff
@@ -0,0 +1,398 @@
+'use client';
+
+import {
+  Article,
+  AccordionProvider,
+  ButtonLink,
+  ButtonGroup,
+  Document,
+  Heading1,
+  Heading2,
+  Heading3,
+  Link,
+  Paragraph,
+  Page,
+  PageContent,
+  SpotlightSection,
+  UnorderedList,
+  UnorderedListItem,
+} from '@utrecht/component-library-react';
+import '@/app/styling/css/detail-page.css';
+import '@utrecht/design-tokens/dist/index.css';
+import { ExampleHeaderFunnel } from '@/components/ExampleHeader/ExampleHeaderFunnel/ExampleHeaderFunnel';
+import { ExampleFooterFocus } from '@/components/ExampleFooter/ExampleFooterFocus/ExampleFooterFocus';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+import { UtrechtDigidLogo, UtrechtIconArrow } from '@utrecht/web-component-library-react';
+import { useForm } from 'react-hook-form';
+import { useRouter } from 'next/navigation';
+import BacklinkIcon from '@/app/styling/assets/backlink-icon.svg';
+
+export default function Home() {
+  const router = useRouter();
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<{ [key: string]: string }>();
+
+  return (
+    <Document>
+      <Page>
+        <ExampleHeaderFunnel />
+        <ExampleNavigation />
+        <PageContent>
+          <Article className="voorbeeld-article-space">
+            <Link className="voorbeeld-link-back" href="./">
+              <BacklinkIcon />
+              <span className="voorbeeld-span--underlined">Terug</span>
+            </Link>
+            <Heading1 className="voorbeeld-heading-1">Verhuizing doorgeven</Heading1>
+            <Paragraph>
+              Als u verhuist naar Utrecht of binnen Utrecht, geeft u uw adres aan ons door. Verhuist u van Utrecht naar
+              een andere gemeente? Geef dan uw verhuizing door in uw nieuwe gemeente.
+            </Paragraph>
+
+            <Link href="https://www.utrecht.nl/city-of-utrecht/moving-to-utrecht-or-moving-within-utrecht">
+              <UtrechtIconArrow />
+              Read this information in English
+            </Link>
+            <SpotlightSection className="voorbeeld-spotlight-section">
+              <Heading2 className="voorbeeld-spotlight-section-heading-style">Verwerking duurt langer</Heading2>
```

Comment:
Is deze class name echt nodig, of kunnen we de code voor `voorbeeld-spotlight-section-heading-style` verplaatsen naar de `.utrecht-spotlight-section` selector?

Reply:
Moet `.utrecht-spotlight-section ` niet met het woord voorbeeld beginnen? 
`voorbeeld-utrecht-spotlight-section` om het consistent te houden en makkelijker maakt om te zoeken? @ryanvvalkenhoef @Robbert

Reply:
alle andere detail page css beginnen met "voorbeeld"